Remotion/Web/ClientScript/Scripts/DropDownMenu.ts

```diff
@@ -0,0 +1,934 @@
+
+      if (DropDownMenu._currentStatusPopup && DropDownMenu._currentStatusPopup === statusPopup)
+      {
+        DropDownMenu.ApplyPosition($(statusPopup), null, titleDivFunc());
+        DropDownMenu._statusPopupRepositionTimer = setTimeout(repositionHandler, DropDownMenu._repositionInterval);
+      }
+    };
+
+    // Only reposition if opened via titleDiv
+    if (evt === null)
+      DropDownMenu._statusPopupRepositionTimer = setTimeout(repositionHandler, DropDownMenu._repositionInterval);
+
+    menuButton.bind('focus', DropDownMenu.FocusHandler);
+    menuButton.bind('blur', DropDownMenu.BlurHandler);
+  }
+
+  private static EndOpenPopUp (menuID: string, context: JQuery, selectionCount: number, evt: Nullable<JQueryMouseEventObject>, itemInfos: DropDownMenu_ItemInfo[]): void
+  {
+    var menuOptionsID = menuID + '_DropDownMenuOptions';
+    var menuButton = $('#' + menuID + ' a[aria-haspopup=menu]');
+
+    if (itemInfos.length == 0)
+      return;
+
+    if (DropDownMenu._statusPopupRepositionTimer)
+      clearTimeout(DropDownMenu._statusPopupRepositionTimer);
+    if (DropDownMenu._currentStatusPopup !== null)
+    {
+      var statusPopup = DropDownMenu._currentStatusPopup;
+      DropDownMenu._currentStatusPopup = null;
+      // Clear the role=alert before removing to item to prevent screenreaders (JAWS) from announcing the old value during removal.
+      statusPopup.removeAttribute('role');
+      statusPopup.parentElement!.removeChild(statusPopup);
+    }
+
+    menuButton.attr('aria-controls', menuOptionsID);
+    menuButton.attr('aria-expanded', 'true');
+
+    var div = document.createElement('div');
+    div.className = 'DropDownMenuOptions';
+    div.id = menuOptionsID;
+    div.setAttribute('role', 'menu');
+    div.setAttribute('tabindex', '-1');
+    if (menuButton.attr('aria-labelledby') !== undefined)
+      div.setAttribute('aria-labelledby', menuButton.attr('aria-labelledby'));
+    else
+      div.setAttribute('aria-labelledby', menuButton[0].id);
+    DropDownMenu._currentPopup = div;
+
+    var ul = document.createElement('ul');
+    ul.className = 'DropDownMenuOptions';
+    ul.setAttribute('role', 'none');
+    div.appendChild(ul);
+
+    const $menuButton = $('#' + menuID);
+    $menuButton.closest('div, td, th, body').append(div);
+
+    $(ul).mouseover (function (event)
+    {
+      var eventTarget = DropDownMenu.GetTarget (event, "LI");
+      $("li", ul).removeClass(DropDownMenu._itemSelectedClassName);
+      if (eventTarget.firstChild != null && eventTarget.firstChild.nodeName.toLowerCase() === 'a')
+      {
+        $(eventTarget).addClass (DropDownMenu._itemSelectedClassName);
+        (eventTarget.firstChild as HTMLElement).focus();
+      }
+    }).keydown (function (event)
+    {
+      // TODO: missing nullcheck?
+      DropDownMenu.Options_OnKeyDown (event, DropDownMenu._currentMenu!);
+    });
+
+    DropDownMenu._itemClickHandler = function(this: HTMLAnchorElement)
```

Comment:
Is this function a declaration error in the original code and should be written as a private instance function with correct casing.